packages/grub-config/config/rec_bootargs.cfg

```diff
@@ -0,0 +1,3 @@
+set kernel=/boot/vmlinuz
```

Comment:
I have already those set of changes in my branch for the squashfs change :+1: I think it's safe to merge this after the squashfs change is done, there are also tests to adjust and scripts (which I'm almost done with) to make everything work with the new format